### PR TITLE
Welsh language toggle

### PR DESCRIPTION
**Pull request by mikeash82**

### CODE REVIEW

File: application/scss/components/_logo.scss [hunk 1]
```diff
@@ -28,3 +28,21 @@
...
+$hmrc-banner-blue: #005ea5;
+
+.hmrc-full-width-banner {
+  display: block;
+  position: relative;
+  z-index: 999;
+  color: govuk-colour("white");
+  background-color: $hmrc-banner-blue;
+
+}
```
Comment (timsb) [inline application/scss/components/_logo.scss:48]: Don't think this is needed either. Could you remove and check the hmrc-banner still works?

File: src/hmrc-design-patterns/welsh-language-toggle/index.njk [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+</ul>
+
+<h2 id="how-it-works" class="govuk-heading-l">How it works</h2>
+
+
+<p class="govuk-body"> Put the Welsh language toggle  on the right hand side below the: </p>
+<ul class="govuk-list govuk-list--bullet">
+  <li><a class="govuk-link" href="https://design-system.service.gov.uk/components/header/">GOV.UK header</a></li>
+  <li> <a class="govuk-link" href="https://design-system.service.gov.uk/components/phase-banner/">phase banner</li>
+  <li> <a class="govuk-link" href="/hmrc-design-patterns/hmrc-banner/">HMRC banner</a> if you are using it</li>
```
Comment (timsb) [inline src/hmrc-design-patterns/welsh-language-toggle/index.njk:33]: Can you make the `href` attribute values consistent - absolute or relative URLs. I'd suggest relative URLs

File: src/hmrc-design-patterns/welsh-language-toggle/index.njk [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+</ul>
+
+<p class="govuk-body"> The Welsh language toggle must be above the <a class="govuk-link" href="https://design-system.service.gov.uk/components/back-link/">back link</a>.</p>
```
Comment (timsb) [inline src/hmrc-design-patterns/welsh-language-toggle/index.njk:36]: Make this relative URL

**Comment by timsb** [comment]
@mikeash82 Could you also remove the Welsh language toggle from the backlog in this repo

File: application/scss/components/_hmrc-internal-header.scss [hunk 4]
```diff
@@ -42,3 +42,11 @@
...
   }
 }
+
+.hrmc-lang-select {
+  @include govuk-font(16);
+  float: right;
+  margin: 1rem 0;
+  width: auto;
+  list-style:none;
+}
```
Comment (timsb) [inline application/scss/components/_hmrc-internal-header.scss:52]: I don't think we need this now. Can you remove please?

Comment (mikeash82) [reply application/scss/components/_hmrc-internal-header.scss:52]: Done

**Comment by timsb** [comment]
Closing as superceded by https://github.com/hmrc/design-system/pull/118

**Pull request closed without merging**